### ISSUE TITLE
Properly implement minecraft dimensions and biomes

### ISSUE THREAD

**Issue by amberkowalski** [feather-rs/feather#401]
# Properly implement minecraft dimensions and biomes

## Description
I am going to implement minecraft's dimension and biome system. This will be useful because we will be able to no longer depend on the NBT file we currently use, and plugins and server owners will be able to adjust biomes / dimensions as they see fit. This includes creating new biomes and dimensions.

**Comment by caelunshun** [comment]
Creating new dimensions will be tricky, as the current `Game` is based on a single world. My vision for multiworld/dimension support in Feather is to use a separate thread and `Game` for each world; communication between these threads can happen through channels.

Since WASM threading is still underway, supporting multiple worlds in plugins isn't really feasible yet. It may be wise to wait until WASM is ready before implementing this.

If you just mean setting the dimension/biome of a single world, then that's a good feature to have right now. But we aren't at a point where multiple worlds can be implemented yet, I think :)

**Comment by amberkowalski** [comment]
Yeah, I have 0 interest in actually implementing the systems to do dimensions. When we are ready to do them, at least we would have this done.
